scripts/OMakefile

```diff
@@ -46,7 +46,7 @@ install:
 	mkdir -p $(DESTDIR)/etc/rc.d/init.d
 	$(IPROG) init.d-xapi $(DESTDIR)$(LIBEXECDIR)/xapi-init
 	$(IPROG) init.d-xapissl $(DESTDIR)$(LIBEXECDIR)/xapissl
-	$(IPROG) init.d-management-interface $(DESTDIR)/etc/rc.d/init.d/management-interface
+	$(IPROG) init.d-management-interface $(DESTDIR)$(LIBEXECDIR)/legacy-management-interface
```

Comment:
Yes we still use it when upgrading from 6.0. xcp-networkd was new in 6.1. Soon we'll be able to purge it altogether when we no longer support that upgrade. It doesn't need a service (in fact we _definitely don't_ want one). There was an autogen one by systemd because it was in `/etc/init.d` which is bad.

It's called directly by xcp-networkd (see the pull-req for that repo that changes the path to the script). Really this script shouldn't be in here but since it's not gonnna be around loads longer I didn't want to take the effort to migrate it to a new repo cos there are some other things that would also need to move (the supporting python modules).